### PR TITLE
chore: add icen as daskom1337 community member

### PR DESCRIPTION
**Pull request by vincentiusantu**

### CODE REVIEW

**Comment by vincentiusantu** [comment]
add icen ascii art
add icen to peoples.js

File: webservice/assets/peoples.js [hunk 1]
```diff
@@ -124,5 +124,17 @@ export function getPeopleList() {
...
+      Handle: "vincentiusantu",
+      Realname: "Vincentius Adisurya Fransisco Antu",
+      Description: `
+      +------------------------------------------------+
+      |         I'm confused what to fill so           |
+      |      put anything here, anything really.       |
+      |                      ~~~                       |
+      |         2022 Academician at Daskom1337         |
+      +------------------------------------------------+`,
+    }
```
Comment (fakhrip) [inline webservice/assets/peoples.js:138]:
```suggestion
      Description: `
+-------------------------------------------------+
|         I'm confused what to fill so            |
|      put anything here, anything really.        |
|                      ~~~                        |
|         2022 Academician at Daskom1337          |
+-------------------------------------------------+`,
    }
```

This one should be changed like this first @vincentiusantu, it wont look correct in the website if it doesnt follow the same box as the other description, change this first, commit and push it and then re-request my review by clicking on the top right corner button in the same place you add me as reviewer before, thanks vincent 👍

**Review by fakhrip — CHANGES_REQUESTED** [review]

File: webservice/assets/peoples.js (hunk 1 above)
Comment (vincentiusantu) [reply webservice/assets/peoples.js]: Done bang

**Review by fakhrip — APPROVED** [review]
Nicee, congratulation for finishing git and github course vincent, and welcome to our community as well! 🎉

**Pull request merged**